Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py

```diff
@@ -285,7 +288,7 @@ def get_file_indicator_reputation(stix_object: dict, reputation_score: int, reli
     Return stix_object of type file as indicator
     """
 
-    hashes = extract_file_indicator_hashes(stix_object.get("pattern"))
+    hashes = extract_file_indicator_hashes(stix_object)
     dbot_score = Common.DBotScore(
         indicator=hashes["md5"],
```

Comment:
What will happen if we don't have the MD5 value (None)? We need to return a value

Reply:
Hello @sapirshuker,

What do you think of:
![image](https://github.com/demisto/content/assets/8960084/2906a45c-5de2-4c5b-93dd-aa44e4cc7dd5)


Have a nice day